BritneyS/GetFed
# Add charts lib

## What you did :question:
- Added Charts library
- Added ability for protein. carbs, and fat food data (macronutrients) to be displayed in food detail view
- Some refactoring

## How you did it :white_check_mark:
- Added Charts library via Cocoapods
- Added PieChartView in stack view
- Set FoodDetailViewController properties 'protein', 'carbs', and 'protein' based on whether or not there was data for those properties in the Food object passed in the segue
- Set those values as entries in a pie chart
- Customized styling and animation of pie chart
- Removed some whitespace for code style purposes


## How to test it :microscope:
- Run the app
- Tap 'Food Search'
- Search for 'apple'
- Select the cell that reads 'Apples, raw, with skin'
- Note that a pie chart that reports the amounts of macronutrients in this selection in grams is animated and displayed

## Screenshots (if applicable) :camera:
![simulator screen shot - iphone xr - 2018-11-29 at 11 38 52](https://user-images.githubusercontent.com/8409475/49237147-a495ee00-f3cb-11e8-94bd-01b6eb566098.png)
